wpilibc/src/main/native/include/frc2/command/RamseteCommand.h

```diff
@@ -0,0 +1,151 @@
+   * and outputs are scaled -1 to 1 for easy consumption by speed controllers.
+   *
+   * <p>Note: The controller will *not* set the outputVolts to zero upon
+   * completion of the path - this is left to the user, since it is not
+   * appropriate for paths with nonstationary endstates.
+   *
+   * @param trajectory                     The trajectory to follow.
+   * @param pose                           A function that supplies the robot
+   * pose - use one of the odometry classes to provide this.
+   * @param follower                       The RAMSETE follower used to follow
+   * the trajectory.
+   * @param ks                             Constant feedforward term for the
+   * robot drive.
+   * @param kv                             Velocity-proportional feedforward
+   * term for the robot drive.
+   * @param ka                             Acceleration-proportional feedforward
+   * term for the robot drive.
+   * @param kinematics                     The kinematics for the robot
+   * drivetrain.
+   * @param leftSpeed                      A function that supplies the speed of
+   * the left side of the robot drive.
+   * @param rightSpeed                     A function that supplies the speed of
+   * the right side of the robot drive.
+   * @param leftController                 The PIDController for the left side
+   * of the robot drive.
+   * @param rightController                The PIDController for the right side
+   * of the robot drive.
+   * @param output                         A function that consumes the computed
+   * left and right outputs (in volts) for the robot drive.
+   * @param requirements                   The subsystems to require.
+   */
+  RamseteCommand(
+      frc::Trajectory trajectory, std::function<frc::Pose2d()> pose,
+      frc::RamseteController follower, units::voltage::volt_t ks,
+      units::unit_t<voltsecondspermeter> kv,
+      units::unit_t<voltsecondssquaredpermeter> ka,
+      frc::DifferentialDriveKinematics kinematics,
+      std::function<units::meters_per_second_t()> leftSpeed,
+      std::function<units::meters_per_second_t()> rightSpeed,
+      frc2::PIDController leftController, frc2::PIDController rightController,
+      std::function<void(units::voltage::volt_t, units::voltage::volt_t)>
+          output,
+      std::initializer_list<Subsystem*> requirements);
+
+  /**
+   * Constructs a new RamseteCommand that, when executed, will follow the
+   * provided trajectory. Performs no PID control and calculates no
+   * feedforwards; outputs are the raw wheel speeds from the RAMSETE controller,
+   * and will need to be converted into a usable form by the user.
+   *
+   * @param trajectory            The trajectory to follow.
+   * @param pose                  A function that supplies the robot pose - use
+   * one of the odometry classes to provide this.
+   * @param follower              The RAMSETE follower used to follow the
```

Comment:
You can call this a controller. Follower is a term popularized by Pathfinder that no one doing controls in academia or industry uses.